openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -896,4 +919,4 @@ def mark_user_change_as_expected(response, new_user_id):
     The new_user_id may be None or an LMS user ID, and may be the same
     as the previous user ID.
     """
-    response.safe_sessions_expected_user_change = {'new_user_id': new_user_id}
+    CONTEXT.data['expected_user_change'] = new_user_id
```

Comment:
Do we want this to be a list? Or do we want any logging or other monitoring if this is not the first user id to be exempted for the request?

Reply:
Might be interesting, but I'm not sure what we'd do with it.